pkgs/by-name/on/onionbalance/package.nix

```diff
@@ -0,0 +1,90 @@
+{
+  fetchFromGitLab,
+  python312Packages,
+  python312,
+  python3,
+  tor,
+  lib,
+  symlinkJoin,
+}:
+
+let
+  src = fetchFromGitLab {
+    domain = "gitlab.torproject.org";
+    owner = "tpo";
+    repo = "onion-services/onionbalance";
+    rev = "2de68025d2ba786936cc9dad67b62360e0a1294b";
+    sha256 = "sha256-Fn62GtA62kkjqHsMmDLIenM5+mxVcVUSS32ADUYGND4=";
+  };
+
+  onionbalance = python312Packages.buildPythonPackage rec {
+    inherit src;
+    pname = "onionbalance";
+    version = "0.2.3";
+    doCheck = false;
+    format = "pyproject";
+    dontCheckRuntimeDeps = true;
```

Comment:
Please explain why. You could also use `pythonRemoveDeps = [ "xx" ]` to remove specific dependencies